### ISSUE TITLE
datasets division

### ISSUE THREAD

**Issue by hexin-666** [IPIC-Lab/IRSAM#9]
Congratulations on your paper being accepted by the European Conference on Computer Vision (ECCV). Your method is really novel and powerful. I have a small question regarding the division of the dataset. I noticed that the ratio of the training set to the test set in the "datasets" directory of your dataset is 8:2, but in the paper, it is 5:3:2.
